std/array.d

```diff
@@ -2021,32 +2045,56 @@ if (isDynamicArray!(E[]) && isForwardRange!R1 && isForwardRange!R2
 {
     assert("Hello Wörld".replace("o Wö", "o Wo") == "Hello World");
     assert("Hello Wörld".replace("l", "h") == "Hehho Wörhd");
+
+    size_t countChanges;
+    assert("Sooo many oooos".replace("o", "ö", countChanges) == "Sööö many öööös");
+    assert(countChanges == 7); // o was replaced 7 times with ö
+}
+
+/// ditto
+void replaceInto(E, Sink, R1, R2)(Sink sink, E subject, R1 from, R2 to)
+{
+    size_t countChanges = 0;
+    replaceInto(sink, subject, from, to, countChanges);
 }
 
 /// ditto
-void replaceInto(E, Sink, R1, R2)(Sink sink, E[] subject, R1 from, R2 to)
```

Comment:
Geez, when was this signature made? The front is the worst place to put the sink argument.